enhancements/network/host-port-registry.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Host Port Registry
+authors:
+  - "@squeed"
+reviewers:
+  - "@dcbw"
+  - "@jsafrane"
+  - "@
+approvers:
+  - "@smarterclayton"
+creation-date: 2020-08-26 
+last-updated: 2020-08-26
+status: implementable
+---
+
+# Host Port Registry
+
+OpenShift has a number of host-level services that listen on a socket. Since they
+are not isolated by network namespace, they can conflict if both listen on the same
+port.
+
+This document serves as the authoritative list of port assignments.
+
+## Background
+
+Ports 9000-9999 are available for general use by system services. We require that
+customers open these ports between nodes in a cluster.
```

Comment:
What about the ports that are not in this range?

Reply:
Two updates here
- there are a few other smaller ranges that are available; wording here needs to mention them. But the installer templates are authoritative
- we're being lazy and using ports outside of this as "effectively localhost" and I need to file bugs to fix this.

Reply:
Note that you can't just change a listener from `0.0.0.0:PORT` to `127.0.0.1:PORT` though, because that will break single-stack IPv6. (`0.0.0.0` is implicitly dual-stack, `127.0.0.1` is not. And `localhost:PORT` is dual-stack for `Dial` but not for `Listen`)